contrib/lite-setup.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2019 FreeIPA Contributors see COPYING for license
+#
+"""In-tree development server setup
+"""
+import argparse
+import os
+import socket
+from urllib.request import urlopen
+
+DEFAULT_CONF = """\
+[global]
+host = {args.hostname}
+server = {args.servername}
+basedn = {args.basedn}
+realm = {args.realm}
+domain = {args.domain}
+# xmlrpc_uri = https://{args.servername}/ipa/xml
+xmlrpc_uri = http://localhost:8888/ipa/xml
+ldap_uri = ldap://{args.servername}
+mode = production
```

Comment:
We don't do significant plugin creation these days but running in development mode is stricter than production mode.

That wouldn't be useful for performance testing though.